### PR TITLE
feat: allow any network to be used in Treasuries

### PR DESCRIPTION
**Pull request by Sekhmet**
### Summary

Now any network should be possible to be used in Treasuries on onchain spaces.

Depends on: https://github.com/snapshot-labs/sx-monorepo/pull/946

Closes: https://github.com/snapshot-labs/sx-monorepo/issues/857

### How to test

1. Go to SX space settings.
2. Create treasuries on some non-SX supported network (Base for example).
3. It shows up in treasuries.

### CODE REVIEW

File: apps/ui/src/components/PickerToken.vue [hunk 1]
```diff
@@ -143,9 +142,9 @@ watch(
       @click="handlePick(asset)"
     >
       <div class="flex items-center min-w-0 pr-2">
-        <UiBadgeNetwork :id="networkId">
+        <UiBadgeNetwork :chain-id="network">
           <UiStamp
-            :id="`${networkId}:${asset.contractAddress}`"
+            :id="`eip155:${network}:${asset.contractAddress}`"
```
Comment (wa0x6e) [inline apps/ui/src/components/PickerToken.vue:147]: This icon seems to not always work

Before this pr, it looks like this

Screenshot 2024-11-05 at 00 32 41

After:

Screenshot 2024-11-05 at 00 32 32

**Review by wa0x6e — CHANGES_REQUESTED** [review]

**Comment by wa0x6e** [comment]
Are we planning another refactoring to rename `network` to `chainId` in relevant places ? We currently have something like 

```
:chain-id="treasury.network"
```

which are not straightforward

**Comment by Sekhmet** [comment]
Yeah, I'd like to refactor it at some point.

**Comment by bonustrack** [comment]
Looks good, there is just these 2 weird networks that appear (it doesn't appear on snapshot.org):
image

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**